### ISSUE TITLE
Error creating safe

### ISSUE THREAD

**Issue by lance-web3** [safe-global/safe-infrastructure#147]
Versions
cgw - v1.50.0 
txs - 5.2.2
cfg - v2.83.0

reference image
![image](https://github.com/safe-global/safe-infrastructure/assets/37203010/fd3407e4-f1f4-4f2e-8096-4ad93c05bb21)

fyi, the error in console it appear before this page, and this page error prompt basically just on the ui and wont log to console.

**Comment by lance-web3** [comment]
need help @moisses89

**Comment by Pana** [comment]
safe issue with https://github.com/safe-global/safe-infrastructure/issues/148

**Comment by moisses89** [comment]
Maybe @safe-global/safe-web can help here.

**Comment by katspaugh** [comment]
Looks like it’s an official deployment on a network that we don’t support. I would recommend debugging what exactly fails. Can’t say much more from the screenshot.

**Comment by lance-web3** [comment]
> Looks like it’s an unofficial deployment on a network that we don’t support. I would recommend debugging what exactly fails. Can’t say much more from the screenshot.

@katspaugh what do you mean network is not supported?

https://github.com/safe-global/safe-core-sdk/pull/901

we have PR both for safe-core-sdk and safe-deployments that is merged.

More ref.
https://github.com/safe-global/safe-core-sdk/issues/909

**Comment by katspaugh** [comment]
I meant from the Wallet side, because @safe-global/safe-web was pinged. The Wallet team only supports the official network and UI deployments on app.safe.global.

As for debugging, try setting breakpoints in the code or inspect the network requests.

**Comment by lance-web3** [comment]
@katspaugh 

can you take a look this, is it safe-wallet-web has to upgrade the protocol-kit version?
https://github.com/safe-global/safe-core-sdk/issues/909#issuecomment-2238526207

**Comment by katspaugh** [comment]
There’s an open pull request that upgrades to v4.

**Comment by lance-web3** [comment]
> There’s an open pull request that upgrades to v4.

ok so I just wait... okay...

**Comment by jerry-nerd** [comment]
@lance-web3 are you running a safe service locally ? I have the same issue.

Is safe-core-sdk and safe-deployments pr must be merged before deploy locally?

**Comment by lance-web3** [comment]
> @lance-web3 are you running a safe service locally ? I have the same issue.
> 
> Is safe-core-sdk and safe-deployments pr must be merged before deploy locally?

I am actually running on public IP, our mainnet chain, but the UI is restrictive only can be accessed by whitelist IP for now

**Comment by lance-web3** [comment]
> There’s an open pull request that upgrades to v4.

@katspaugh I see the PR that upgrades to v4 got merged, but its v4.0.1 not v4.0.3 of protocol-kit

**Comment by lance-web3** [comment]
fixed after pulled newer version of components today, current:
```
safe-transaction-service - v5.7.0
safe-client-gateway - v1.52.0
safe-config-service - v2.83.0
safe-wallet-web - v1.40.1
```
